### PR TITLE
Hotfix/affiliate disclaimer

### PR DESCRIPTION
**Pull request by mekarpeles**
Closes request for visible affiliate disclosure

Test here:
https://dev.openlibrary.org/books/OL15626440M/Problem-based_learning

- bwb links should still resolve
- disclosure message should appear in vendors box

dev openlibrary org_books_OL1409749M_A_Night_in_the_Lonesome_October

### Stakeholders

@bfalling

### CODE REVIEW

File: openlibrary/macros/AffiliateLinks.html [hunk 1]
```diff
@@ -64,4 +63,6 @@
         $for store in more_stores:
           $:affiliate_link(store['key'], store['analytics_key'], store['name'], store['link'], store.get('price', ''), store.get('price_note', ''))
       </details>
+    
+    <p>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</p>
```
Comment (cdrini) [inline openlibrary/macros/AffiliateLinks.html:67]:
```suggestion
    <small>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</small>
```

**Review by cdrini — APPROVED** [review]
One small comment for better semantic HTML

**Review by bfalling — APPROVED** [review]
LGTM

File: openlibrary/macros/AffiliateLinks.html [hunk 2]
```diff
@@ -64,4 +63,6 @@
         $for store in more_stores:
           $:affiliate_link(store['key'], store['analytics_key'], store['name'], store['link'], store.get('price', ''), store.get('price_note', ''))
       </details>
+    
+    <small>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</small>
```
Comment (mekarpeles) [inline openlibrary/macros/AffiliateLinks.html:67]: `small` breaks the styling and I'd rather not add more css for a small semantic victory at this stage. Happy to accept a followup pr to anyone who wants to fix.

```suggestion
    <p>When you buy books using these links the Internet Archive may earn a <a class="nostyle" href="/help/faq/about#selling">small commission</a>.</p>
```

**Pull request merged**